### PR TITLE
fix: allow copying the results of callPrivateRPC

### PR DESCRIPTION
**Pull request by richard-ramos**
Also allows selecting text in the input used to write the RPC request

### CODE REVIEW

**Review by jrainville — APPROVED** [review]

**Review by borismelnik — APPROVED** [review]

File: ui/app/AppLayouts/Node/NodeLayout.qml [hunk 1]
```diff
@@ -185,7 +186,14 @@ StatusSectionLayout {
             Layout.leftMargin: Style.current.padding
             // TODO: replace with StatusTextArea once it lives in StatusQ.
             // Not Refactored Yet
```
Comment (caybro) [inline ui/app/AppLayouts/Node/NodeLayout.qml:188]: You can drop the TODOs :)

File: ui/app/AppLayouts/Node/NodeLayout.qml [hunk 2]
```diff
@@ -185,7 +186,14 @@ StatusSectionLayout {
...
-            TextArea { id: callResult; Layout.fillWidth: true; text: root.store.nodeModelInst.callResult; readOnly: true }
+            StatusTextArea { 
+                id: callResult;
+                Layout.fillWidth: true;
+                text: root.store.nodeModelInst.callResult;
```
Comment (caybro) [inline ui/app/AppLayouts/Node/NodeLayout.qml:192]: trailing semicolons

File: ui/app/AppLayouts/Node/NodeLayout.qml [hunk 3]
```diff
@@ -185,7 +186,14 @@ StatusSectionLayout {
...
+                readOnly: true
+                selectByMouse: true
```
Comment (caybro) [inline ui/app/AppLayouts/Node/NodeLayout.qml:194]: This is the default

File: ui/app/AppLayouts/Node/NodeLayout.qml [hunk 4]
```diff
@@ -185,7 +186,14 @@ StatusSectionLayout {
...
+                wrapMode: TextEdit.WordWrap
```
Comment (caybro) [inline ui/app/AppLayouts/Node/NodeLayout.qml:195]: Default as well

**Review by caybro — COMMENTED** [review]
Minor stuff otherwise LGTM

**Pull request merged**